TileStache/Goodies/VecTiles/transform.py

```diff
@@ -346,6 +346,30 @@ def water_tunnel(shape, properties, fid, zoom):
     return shape, properties, fid
 
 
+boundary_admin_level_mapping = {
+    2: 'country',
+    4: 'state',
+    6: 'county',
```

Comment:
The mapping of admin level integer to what it represents [is pretty complex and country-specific](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dadministrative#10_admin_level_values_for_specific_countries), but I'd recommend adding `8: 'municipality'`, as we might find that comes in useful later.